support/runtime-upgrade/example_mnemonic

```diff
@@ -0,0 +1 @@
+//Alice
```

Comment:
This isn't a mnemonic. The script takes any Substrate private key (mnemonic, derivation path, etc). The argument and docs including 'mnemonic' could make users think they needed to provide a mnemonic, when any key would work.

I'd change any reference to 'mnemonic' to privateKey or something. Also this file should have an extension, something like `path/key.priv`.